package.json

```diff
@@ -73,7 +73,7 @@
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-testing-library": "^5.9.1",
     "fs-extra": "10.1.0",
-    "husky": "^8.0.0",
+    "husky": "^8.0.3",
```

Comment:
Pretty sure this isn't needed for your change, if you undo this then you'd avoid changes to the `yarn.lock` as well

Reply:
No worries!